src/entity/Session.ts

```diff
@@ -10,8 +10,11 @@ export class Session {
 	hashed_uuid: string;
 
 	@Column()
-	ended_at: Date;
+	start: Date;
 
-	@OneToMany(() => Action, action => action.session)
-	actions: Action[];
+	@Column()
+	end: Date;
+
+	@OneToMany(() => Event, action => action.session)
+	actions: Event[];
```

Comment:
should probably be named events

Reply:
You're not meant to use plural versions of names in databases

Reply:
I meant the field "actions"

Reply:
Oh right, that probably got missed when I mass renamed the value